### ISSUE TITLE
Change Interval

### ISSUE THREAD

**Issue by acaliaro** [winstongubantes/MatchaBackgroundService#13]
Hi guys.

I would like to change the "Interval" during runtime. Is it possible?

It seems it has always the "default" value

 ```
  public class PrintPdfTask : IPeriodicTask
    {
        public PrintPdfTask()
        {
        }

        public TimeSpan Interval { get; set; } = TimeSpan.FromSeconds( Preferences.FTP_POLLING); 

        public async Task<bool> StartJob()
        {
            // YOUR CODE HERE
            // THIS CODE WILL BE EXECUTE EVERY INTERVAL
            System.Diagnostics.Debug.WriteLine("StartJob. Interval: " + Interval.ToString());
            bool doWork = true;

            try
            {
                // Prendo il nuovo Interval
                try
                {
                    int seconds = int.Parse(Preferences.GetFtpPolling());
                    if (seconds > 0)
                        **Interval = TimeSpan.FromSeconds(seconds);**
                    else
                        doWork = false;
                }
                catch
                {
                    doWork = false;
                }

#if DEBUG
                doWork = false;
#endif

                // Il polling è sempre attivo. Se nella preferences non ho il valore, disabilito l'esecuzione del codice
                if(doWork)
                    await Print.PrintPdf(false);
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("PrintPdf: " + ex.Message);
            }

            return true; //return false when you want to stop or trigger only once
        }
    }
```

**Comment by nguyenthanhliemfc** [comment]
Do you have try 
`BackgroundAggregatorService.Add(() => new PrintPdfTask(4)); //4 is seconds`

**Comment by acaliaro** [comment]
Hi. No I haven't tried because the Add should "add" another task… or does it change an already existing task?

**Comment by winstongubantes** [comment]
Hi @acaliaro we will be supporting dynamic interval in the next release. stay tuned.

**Comment by winstongubantes** [comment]
Hi @acaliaro , we decided not to have the dynamic Interval it does cause weird behavior on the Service itself. Please try this workaround => https://www.screencast.com/t/vpRUsedC4
